stdlib/logging/config.pyi

```diff
@@ -15,9 +33,65 @@ if sys.version_info >= (3, 7):
 else:
     _Path = StrPath
 
+_Tkey = TypeVar("_Tkey", bound=Hashable)
+_TValue = TypeVar("_TValue")
+_T = TypeVar("_T")
+_V = TypeVar("_V")
+
 DEFAULT_LOGGING_CONFIG_PORT: int
 RESET_ERROR: int  # undocumented
 IDENTIFIER: Pattern[str]  # undocumented
+_listener: Optional[Any]
+dictConfigClass = DictConfigurator
+
+def _resolve(name: str) -> _V: ...
+def _strip_spaces(alist: List[str]) -> Iterable[_Tkey]: ...
+def _create_formatters(cp: Dict[_Tkey, Dict[_Tkey, _TValue]]) -> Dict[_Tkey, _TValue]: ...
+def _install_handlers(
+    cp: Dict[_Tkey, Dict[_Tkey, _TValue]], formatters: Dict[_Tkey, Dict[_Tkey, _TValue]]
+) -> Dict[_Tkey, _TValue]: ...
+def _handle_existing_loggers(
+    existing: List[Union[str, Logger]], child_loggers: List[Union[str, Logger]], disable_existing: bool
+) -> None: ...
+def _install_loggers(cp: Dict[_Tkey, Dict[_Tkey, _TValue]], handlers: Dict[_Tkey, _TValue], disable_existing: bool) -> None: ...
+def _clearExistingHandlers() -> None: ...
```

Comment:
Our convention is to not add `_non_public` members, unless you have a specific need for a specific `_non_public` member.